frontend/src/utils.js

```diff
@@ -43,3 +48,16 @@ export function getTagColor(tags: Map<string, string>, tag: string): string {
   const color: ?string = tags.get(tag);
   return color ? color : '#bbbbbb';
 }
+
+export function byAlphabetical(a, b) {
+  var nameA = a.name.toLowerCase();
+  var nameB = b.name.toLowerCase();
+
+  if (nameA < nameB) {
+    return -1;
+  } else if (nameA > nameB) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
```

Comment:
Just a typo: please keep a new line at the end of the files.
It's a good practice as it can help with command line tools.